apps/fluent-tester/src/FluentTester/theme/applyBrand.ts

```diff
@@ -1,5 +1,21 @@
 export type OfficeBrand = 'Default' | 'Office' | 'Word' | 'Excel' | 'Powerpoint' | 'Outlook';
-type BrandRampKey = 'App1' | 'App2' | 'App3' | 'App4' | 'App5' | 'App6' | 'App7' | 'App8';
+type BrandRampKey =
+  | 'App1'
+  | 'App2'
+  | 'App3'
+  | 'App4'
+  | 'App5'
+  | 'App6'
+  | 'App7'
+  | 'App8'
```

Comment:
I had assumed these were the same as Primary/Tint/Shade that you had below (just with more confusing names). Are these values used somewhere?

Reply:
OK I see that you have completely different hex values. What does App1-8 semantically mean?

Reply:
So App1-8 are the old brand color names from Office. They're being replaced by the AppTint-AppShade colors.

Since the older entries are based on the old App colors, I elected to just add them instead of ripping it out